packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -702,7 +732,11 @@ function completeAliasName({
     candidateRule.ruleList.includes(CypherParser.RULE_createAlias) &&
     candidateRule.ruleList.includes(CypherParser.RULE_aliasName)
   ) {
-    return parameterSuggestions;
+    return parameterSuggestions.map((completionItem) => ({
+      label: completionItem.label,
+      kind: completionItem.kind,
+      insertText: backtickParamIfNeeded(completionItem.label),
+    }));
```

Comment:
We should move the insertText thing to where the parameterSuggestions are defined instead of repeating this in the code a few times